Detections/Syslog/squid_cryptomining_pools.txt

```diff
@@ -0,0 +1,66 @@
+// Name: Squid proxy events for ToR proxies 
+//
+// Id: 90d3f6ec-80fb-48e0-9937-2c70c9df9bad
+//
+// Description: Check for squid proxy events associated with common ToR proxies. This query presumes the default squid log format is being used.
+//
+// DataSource: #Syslog
```

Comment:
Minor change, but we want to go forward with a different format.  We want to include the DataConnector and the DataType, example:
DataConnector: #AzureActiveDirectory, #Office365; DataTypes: #SigninLogs, #OfficeActivity

For all of yours, let's go with - DataConnector: #Syslog; DataTypes: #Squid